bitbots_head_behavior/src/bitbots_head_behavior/actions/search_pattern.py

```diff
@@ -94,19 +94,23 @@ def perform(self, reevaluate=False):
         head_tilt = math.radians(head_tilt)
         rospy.logdebug("Searching at {}, {}".format(head_pan, head_tilt))
 
-        self.blackboard.head_capsule.send_motor_goals(
+        success = self.blackboard.head_capsule.send_motor_goals(
             head_pan,
             head_tilt,
             pan_speed=self.pan_speed,
             tilt_speed=self.tilt_speed,
             current_pan_position=current_head_pan,
             current_tilt_position=current_head_tilt)
 
-        distance = math.sqrt((current_head_pan - head_pan) ** 2 + (current_head_tilt - head_tilt) ** 2)
+        if not success:
+            # Try the next pattern position
```

Comment:
I case of the pattern a warning should be printed, because the pattern config is bad if they result in a collision